### DISCUSSION TITLE
How to open MyOldProject.git folder? Thanks

### DISCUSSION THREAD

**Discussion by Emin-dev** [community/community#23881] [New to GitHub]
<p>how to read Exported account data github?</p>
<p>how to open exported account data github?</p>
<p>after exporting user data i cant open ( file.tar.gz   &gt;   repositories    &gt;   myUserName   &gt;  MyOldProject.git folder )</p>
<p>and now all my projects is folderName.git    why?</p>
<p>and how can i open read them as normal  html css js …</p>
<p>please hellppp)</p>

**Comment by airtower-luna** [comment]
<p>Without having seen your data, a directory named something like <code>repository.git</code> is most likely a <a href="https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefbarerepositoryabarerepository"><em>bare repository</em></a>. You can just clone it to get a copy with a working tree, e.g. <code>git clone /path/to/bare/repo.git</code>.</p>

**Comment by Emin-dev** [comment]
<p>Sorry but how?</p>
<p>git clone /path/to/bare/repo.git</p>
<p>i tried this but doesn’t work</p>
<p>$ git clone /path/to/bare/Portfolio.git<br>
Cloning into ‘Portfolio’…<br>
fatal: ‘C:/Program Files/Git/path/to/bare/Portfolio.git’ does not appear to be a git repository<br>
fatal: Could not read from remote repository.</p>
<p>Please make sure you have the correct access rights<br>
and the repository exists.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>You need to replace the <code>/path/to/bare/</code> with the actual path to the repository folder on your system, that’s just a placeholder to show you need to give the path. You could also run e.g. <code>git log</code> inside the <code>Portfolio.git</code> folder to confirm my assumption that it’s a bare repository, if you get a useful log it is. :slightly_smiling_face:</p>

**Comment by Emin-dev** [comment]
<p>Thanks,  yes it works</p>
